app.py

```diff
@@ -108,5 +100,14 @@ def compare_members(ldap_group, github_team, attribute='username'):
     return sync_state
 
 
-def sync_team():
-    pass
\ No newline at end of file
+def sync_team(org, team ,state):
```

Comment:
I didn't even notice at first but `sync_team` gets redefined here.  think we need to pick another name